editor/example/index.ts

```diff
@@ -53,6 +54,11 @@ export const examples: {
         name: 'Basic Example: Rule Mark',
         spec: EX_SPEC_RULE
     },
+    MATRIX: {
+        name: 'Basic Example: Hi-C Matrix',
+        spec: EX_SPEC_MATRIX,
+        forceShow: true
```

Comment:
```suggestion
        forceShow: false
```

I will turn this off after the review so that our public editor does not directly show this example.